### PR TITLE
Polymorphism documentation and Sample Codes created

### PR DESCRIPTION
**Pull request by Chamika36**
Polymorphism documentation and sample codes of method overloading and method overriding were added.

### CODE REVIEW

File: 1. Object Oriented Programming/2. Polymorphism/1. Introduction/MethodOverloadSample.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+	int add(int a , int b ) {
+		return a+b;
+	}
+	
+	int add(int a , int b , int c ) {
+		return a+b;
+	}
+
+}
```
Comment (buddhikac96) [inline 1. Object Oriented Programming/2. Polymorphism/1. Introduction/MethodOverloadSample.java]: Add an empty line to the end of the file

File: 1. Object Oriented Programming/2. Polymorphism/1. Introduction/MethodOverrideSample.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		System.out.println("moow");
+	}
+}
+
+class Cat extends Animal{
+	@Override
+	void makeSound() {
+		System.out.println("meow");
+	}
+}
```
Comment (buddhikac96) [inline 1. Object Oriented Programming/2. Polymorphism/1. Introduction/MethodOverrideSample.java]: Add an empty line to the end of the file

File: 1. Object Oriented Programming/2. Polymorphism/1. Introduction/Doc.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Compile time polymorphism 
+- Runtime polymorphism 
+
+### Compile time polymorphism 
+
+Method overloading - Same class having methods with the same name but with different Parameters.
+
+### Runtime polymorphism 
+
+Method overriding - A subclass overrides a method with the same signature as that of in its superclass.
```
Comment (buddhikac96) [inline 1. Object Oriented Programming/2. Polymorphism/1. Introduction/Doc.md]: Please use your code also to explain the concept

Comment (Chamika36) [reply 1. Object Oriented Programming/2. Polymorphism/1. Introduction/Doc.md]: I updated the code and the documentation as required. Can you check them please ?

**Comment by buddhikac96** [comment]
@Chamika36 This is mergeable now. Thanx for the code!

**Comment by Chamika36** [comment]
> @Chamika36 This is mergeable now. Thanx for the code!

Thankyou 🖤

**Pull request merged**